### PR TITLE
Update MacroDefs.swift for windows compat 

### PR DESCRIPTION
**Pull request by Matt-Is-Confused**
Proposed change for #214
Needs to be tested on windows

### CODE REVIEW

File: Sources/SwiftGodot/MacroDefs.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //  Created by Miguel de Icaza on 9/25/23.
 //
 
-#if !os(Windows)
+#if !(os(Windows) && swift(>=5.9))
```
Comment (tishin) [inline Sources/SwiftGodot/MacroDefs.swift:8]: This will enable macros for Windows with Swift < 5.9

Comment (Matt-Is-Confused) [reply Sources/SwiftGodot/MacroDefs.swift:8]: good catch I got myself mixed up

File: Sources/SwiftGodot/MacroDefs.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@
 //  Created by Miguel de Icaza on 9/25/23.
 //
 
-#if !(os(Windows) && swift(>=5.9))
+#if !(os(Windows) && swift(<5.9))
```
Comment (tishin) [inline Sources/SwiftGodot/MacroDefs.swift]: This will still leave the macros enabled for swift 5.9 which doesn't have macro support. It should be <=5.9.

**Comment by Matt-Is-Confused** [comment]
This still needs to be tested on a windows install of swift with both 5.9 and 5.9.1 to ensure it doesn't ignore the minor version in the check

**Comment by migueldeicaza** [comment]
Thanks for the contribution!

**Pull request merged**